packages/manager/modules/product-offers/README.md

```diff
@@ -71,6 +71,8 @@ export default class {
       getPlanCode: this.getPlanCode.bind(this),
       productName: 'webHosting',
       serviceNameToAddProduct: 'nameOfMyProduct',
+      onPricingSubmit: () => console.log('onPricingSubmit'),
+      onValidateSubmit: () => console.log('onValidateSubmit'),
```

Comment:
```suggestion

```

Reply:
It is in the README file ;), to display the possibility to use those two methods